docs/index.md

```diff
@@ -1,3 +1,7 @@
+---
+order: 1
+---
+
 # Getting started
```

Comment:
Please update the link in `README.md` to point here. At least until we can deploy the new site 🙃